tests/improver-nbhood/02-basic.bats

```diff
@@ -46,3 +46,20 @@
   rm "$TEST_DIR/output.nc"
   rmdir "$TEST_DIR"
 }
+
+@test "nbhood 'square' --radius-in-km=20 input output" {
```

Comment:
What happens if one test passes and one test fails? I'm guessing it might be hard to de-bug. But you're right, if we did move square neighbourhood test then the basic test would need renaming.